packages/forms/resources/views/components/field-wrapper/index.blade.php

```diff
@@ -39,7 +39,11 @@
 
                 @if ($hint || $hintIcon)
                     <x-forms::field-wrapper.hint :icon="$hintIcon">
-                        {{ filled($hint) ? ($hint instanceof \Illuminate\Support\HtmlString ? $hint : \Illuminate\Support\Str::of($hint)->markdown()->sanitizeHtml()->toHtmlString()) : null }}
+                        @if ($hint instanceof \Illuminate\Support\HtmlString || $hint instanceof \Filament\Forms\Components\Actions\Action)
+                            {{ $hint }}
+                        @else
+                            {{ \Illuminate\Support\Str::of($hint)->markdown()->sanitizeHtml()->toHtmlString() }}
```

Comment:
We should probably also handle `$hint === null` to prevent the stringable stuff in that case for performance?

Reply:
It's handled in the preceding `@if ($hint || $hintIcon)`

Reply:
Not always, when `$hintIcon` is set and `$hint` is `null`, that condition passes and thus `$hint` gets passed to `Str::of()`.

Reply:
Ok